hello-parsec.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+
+# Copyright 2022 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+ping_parsec() {
+    echo "Checking for availability of the Parsec service on your system... "
+    $PARSEC_TOOL ping > /dev/null
+}
+
+PARSEC_SERVICE_ENDPOINT="${PARSEC_SERVICE_ENDPOINT:-unix:/run/parsec/parsec.sock}"
+PARSEC_TOOL="${PARSEC_TOOL:-$(which parsec-tool)}"
+KEY_NAME=HelloParsecDemoKey
+export RUST_LOG="${RUST_LOG:-error}"
+
+if ! ping_parsec; then exit 1; fi
```

Comment:
if ping parsec is not successful, then shall we print an error message here? e.g.
`Parsec not configured correctly on this system.`